### PR TITLE
Implemented Add Comment dialog to tag suspect values

### PR DESCRIPTION
**Pull request by dannyparsons**
@africanmathsinitiative/developers this is ready for review

There isn't a specific issue but relates to as mentioned in #5729

The Add Comment dialog is accessible from the row or cell right click menus and should autofill the selected row and column.

### CODE REVIEW

**Review by rdstern — APPROVED** [review]
@dannyparsons brilliant.  

There are lots to consider adding including:
a) Clear button on the dialogue to clear the text.  
b) Add the facility to have a comment on the Column and also on the data frame
c) Could have the right click Add Comment on the main part of the dialogue.  If in the column, then for that.  In a cell, then for the cell.

But all that's for later.

**Review by lloyddewit — DISMISSED** [review]
@dannyparsons Just a couple of optional small comments

File: instat/dlgAddComment.vb [hunk 1]
```diff
@@ -16,11 +16,13 @@
 
 Imports instat.Translations
 Public Class dlgAddComment
-    Public bUseSelectedColumn As Boolean = False
```
[the remark this answers is not in the document]
Comment (dannyparsons) [reply instat/dlgAddComment.vb:19]: Thanks done.

**Comment by dannyparsons** [comment]
@lloyddewit I've addressed those comments.

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern just a few small changes since you last approval that should not have changed the functionality, please could you approve again? thanks

**Review by rdstern — APPROVED** [review]

**Pull request merged**